ci-operator/step-registry/interop/mtr/execute/interop-mtr-execute-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Set variables used to update the config file
```

Comment:
why not use jinja template for config files?

Reply:
I will consider that when this is refactored. PQE is re-factoring their tests into a new repository, so this may not be needed anymore.